superset-frontend/packages/superset-ui-core/test/query/extractExtras.test.ts

```diff
@@ -17,12 +17,13 @@
  * under the License.
  */
 import extractExtras from '../../src/query/extractExtras';
+import { TimeGranularity } from '../../src/time-format/types';
 
 describe('extractExtras', () => {
   const baseQueryFormData = {
     datasource: '1__table',
     granularity_sqla: 'ds',
-    time_grain_sqla: 'PT1M',
+    time_grain_sqla: TimeGranularity.MINUTE,
```

Comment:
same here re the import

Reply:
Is this one also fine for now, pending more opinions?

Reply:
I would remove these two imports and stay with the strings for now.

Reply:
I ran into linting errors when I had them as just strings. But that was several iterations ago, can change back and see if linting is ok

Reply:
this looks better!